File: official/docs/node/trackers/create_tracker.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

const tracker = new api.Tracker({
  tracking_code: 'EZ1000000001',
  carrier: 'USPS',
});

tracker.save().then(console.log);

File: official/docs/csharp/trackers/CreateTracker.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task CreateTracker()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        Tracker tracker = await Tracker.Create("USPS", "9400110898825022579493");

        new TestOutputHelper().WriteLine(JsonConvert.SerializeObject(tracker, Formatting.Indented));
    }
}
